src/ChannelServer/Util/GmCommands.cs

```diff
@@ -488,7 +489,16 @@ private CommandResult HandleClearInventory(ChannelConnection conn, Character sen
 
 			return CommandResult.Okay;
 		}
-	}
+
+		private CommandResult HandleTestHealSkill(ChannelConnection conn, Character sender, Character target, string command, string[] args)
```

Comment:
I left it there, to test Ground skills. I could change that to TestGroundSkill() in the future.